Файлы objects.py и enemies.py не задействованы 